src/components/02_atoms/Widgets/TextTextarea.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import RichTextEditor from 'react-rte';
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+let styles;
+
+/**
+ * Some basic wysiwyg editor based upon react-rte which is based upon draft.js.
+ * This was designed to be as easiest as possible for the demo.
+ *
+ * On the longrun we might want to switch back to ckeditor.
+ */
+class TextTextarea extends Component {
```

Comment:
I think we use `React.Component` in other places?

Reply:
👍